### ISSUE TITLE
[EN][Death Knight] armor

### ISSUE THREAD

**Issue by stanq87** [Atlantiss/BugTracker#9876]
**Description**:  blood dk tank seems to get 80,459% insted of expected 69%
armor from items = 22696, expected 38356,24  but we get 40957

toughness +10%   http://cata-shoot.tauri.hu/?spell=49787
Rune of the Stoneskin Gargoyle +4% http://cata-shoot.tauri.hu/?spell=62158
bloodpresence +55%  http://cata-shoot.tauri.hu/?spell=48263

**Current behaviour**: we get 180,459% armor bonus

**Expected behaviour**: we should get 169%

**Steps to reproduce the problem**:

1. be bd dk
2. sum armor
3. have  toughness(talent), Rune of the Stoneskin Gargoyle weapon enchant, be in blood presence
4. look how much armor u get
5. compare 

**Revision**: rev. 5637

**Comment by stanq87** [comment]
ps sometimes when unequip all your armor u still have some armor, not sure if visual or what

**Comment by Psychobird7** [comment]
These effects are not additive,  

You also did not bring any evidence of the expected behaviour.

**Comment by stanq87** [comment]
OK fine so lets start

1. no talents no presence  items should give me 22696 armor  (3057+2822+769+120+3762+1675+2393+2116+3350+2632), BUT the game sums this into 23147, this is the first bug, unless adding in wow has dimishing resturn or something , we are off by  451 armor

2. now, adding the next +10% (toughness) from 23147 gave me  25461,7 ( in game  25450, a bit off but more or less what was expected) (if the game would have summmed corectly the armor value of items we would have 24965,6 now, not 25461,7 or 25450)

3. next is presence +55%,  so taking the 3 posibilities of armor the game should give us :
24965,6 +55%, 38696,68
25461,7 +55%, 39465,635
25450 +55%, 39447,5
but we are geting 39381 ingame

in conclusion  the server is bad at adding and multiplaing posiibly also at dividing 
folowing the same in stps game on paper we get  slight errors  even  we abandon  the numbers we got and used the ones the game provided

so: we expected to get   38696,68 from the base  22696 armor
we got 39381 from 23147 in game  ( which is 170% of the base or  154% after 10% from talents, using the wrong armor sum)

**Comment by Katonetwo** [comment]
armor is fine, this is silly

**Comment by stanq87** [comment]
how can it be fine when the server dosent even handle summing the armor from items ?

**Comment by Katonetwo** [comment]
the armor enchant on your cloak is unaffected by toughness, blood presence or the armor meta
using the numbers you gave here:

"no talents no presence items should give me 22696 armor (3057+2822+769+120+3762+1675+2393+2116+3350+2632), BUT the game sums this into 23147, this is the first bug, unless adding in wow has dimishing resturn or something , we are off by 451 armor"

your cloak enchant gives you 120 armor, so let's subtract that from 22696
that gives us 22576
multiply by 1.02 due to the armor meta, we get 23027
add the flat 120 from the cloak enchant
that puts us at exactly 23147

the server calculates armor perfectly fine
it's you that made the mistake

**Comment by stanq87** [comment]
yep damn meta, forgot abaut it, recheking stuff, but so far it adds up corectly,

**Comment by stanq87** [comment]
@Psychobird7 

> The other options are increasing our passive reduction (considering we are *way* behind on passive reduction this is the most likely result IMO). We can go back to the 60% armor we had back in WotLK, which would provide another ~10k armor and put is just slightly over where our block friends are (We'd be around 43k unbuffed where they are running around with ~39-40k unbuffed and net us about 6% more reduction). They *could* further increase the passive damage reduction from blood presence, but that would impact magic abilities and we are already strong enough at handling those. They could buff Bone Shield to be a solid passive damage reduction, but this removes a semi-powerful cooldown from us that all other tanks have and just puts us up to par passively with other tanks, so it's a near nerf.

http://web.archive.org/web/20120126104516/http://elitistjerks.com:80/f72/t125290-blood_dk_endgame_tanking_4_x

meantime, dktank have about 2-3k more armor on atlantis then paladins/tank is same gear

**Comment by Psychobird7** [comment]
That screenshot is from 4.2 you fucknugget. Blood DK got buffed in 4.3.

**Comment by stanq87** [comment]
well if its 4.2 content and pre buff 30-->55% armor then his items give him 22159
and after folowing
22159x1,1x1,04x1,3x1,02 we coma at 37639
but if we follow
22159x(1+0,04+0,3+0,02) we have 36377
plus all the posible raid buff , ofcourse , assuming he has the gem the wepoan enchants itp etc
the same dk after the boost in raid would have 44k armor ?
